libmproxy/protocol2/auto.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import (absolute_import, print_function, division, unicode_literals)
+from .layer import Layer
+
+
+class AutoLayer(Layer):
+    def __call__(self):
+        d = self.client_conn.rfile.peek(1)
+        if d[0] == "\x16":
```

Comment:
I guess this is a SSL/TLS handshake magic value defined in the spec?
I don't know the actual SSL/TLS handshake on the wire - so this really is magic for me.
Maybe a short comment about it would help...